### PR TITLE
enable loading from 8 parts

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,9 +15,15 @@
...
+    "ip",
+    "ipv4",
+    "ipv6"
+  ],
   "repository": "git://github.com/whitequark/ipaddr.js",
   "main": "./lib/ipaddr",
-  "engines": { "node": ">= 0.10" },
+  "engines": {
+    "node": ">= 0.10"
+  },
```
Comment (whitequark) [inline package.json]: Please avoid gratuitous reformatting.

File: ipaddr.min.js [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+lib,ipaddr.js;
```
Comment (whitequark) [inline ipaddr.min.js]: This is broken.

File: package.json (hunk 1 above)
Comment (nopcoder) [inline package.json]: sure, I think it was yarn

**Comment by whitequark** [comment]
On reflection, a function called <code>ipaddr.from**Byte**Array</code> should not accept an array of 16-bit values, so I won't be merging this.

**Comment by nopcoder** [comment]
OK, thanks.

**Pull request closed without merging**